thoth/common/openshift.py

```diff
@@ -1516,9 +1516,13 @@ def schedule_kebechet_workflow(
     ) -> Optional[str]:
         """Schedule Kebechet Workflow for a Webhook from GitHub App.."""
         workflow_id = job_id or self.generate_id("kebechet-job")
+        url = webhook_payload["repository"]["html_url"]
```

Comment:
are you looking for pull_request.base.repo.full_name?

Reply:
Ummm, not sure what `pull_requrest.base.repo.full_name` is. I'm looking for `https://github.com/<foo>/<bar>` and then I am trying to extract `<foo>/<bar>`

Reply:
the webhook_payload["repository"]["full_name"] might be just enough as @goern suggested.

Reply:
ja, right :)